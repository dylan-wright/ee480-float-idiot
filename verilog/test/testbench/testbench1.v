op = `OPi2f;
x = 0;
#1 $display("%h", z);
x = 1;
#1 $display("%h", z);
x = 2;
#1 $display("%h", z);
x = 3;
#1 $display("%h", z);
x = 4;
#1 $display("%h", z);
x = 5;
#1 $display("%h", z);
x = 6;
#1 $display("%h", z);
