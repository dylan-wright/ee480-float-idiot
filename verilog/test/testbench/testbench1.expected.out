0000
3f80
4000
4040
4080
40a0
40c0
